src/util/pg.py

```diff
@@ -14,6 +14,8 @@
 from functools import partial, reduce
 from multiprocessing import cpu_count
 
+from .exceptions import UpgradeError
```

Comment:
Do it below, with the other imports from `.exceptions`

Reply:
Oops, Already imported `MigrationError`
Done